app/styles/_login.scss

```diff
@@ -188,3 +188,30 @@
     }
   }
 }
+
+.ember-modal-dialog{
+  padding: 30px 40px;
+  width: 42%;
+    form input{
```

Comment:
Can you fix all the indentation here? It's hard to read when it's inconsistent.

Reply:
WIll fix accordingly.